modules/ROOT/pages/create-endpoint.adoc

```diff
@@ -35,7 +35,7 @@ For more information about creating endpoints from the message flow wizard, see
 
 == Supported Endpoint Protocols
```

Comment:
This topic looks like info the audience should have before performing the tasks of creating and configuring and likely belongs in a Concept topic elsewhere, maybe the one describing send and receive endpoints?

Reply:
The PM wanted this table here and a note in each of the topics with the correct information. 

I agree with you and discussed this already, but this is where he wants it to be.